shardingsphere-test/shardingsphere-rewrite-test/src/test/resources/scenario/sharding/case/select.xml

```diff
@@ -74,22 +74,25 @@
 
     <rewrite-assertion id="select_with_subquery_and_different_sharding_conditions" db-types="MySQL">
         <input sql="SELECT * FROM (select t_account.account_id from t_account where t_account.account_id=?) a WHERE account_id = 101" parameters="100" />
-        <output sql="SELECT * FROM (select t_account.account_id from t_account where t_account.account_id=?) a WHERE account_id = 101" parameters="100" />
+        <output sql="SELECT * FROM (select t_account_0.account_id from t_account_0 where t_account_0.account_id=?) a WHERE account_id = 101" parameters="100" />
+        <output sql="SELECT * FROM (select t_account_1.account_id from t_account_1 where t_account_1.account_id=?) a WHERE account_id = 101" parameters="100" />
     </rewrite-assertion>
 
     <rewrite-assertion id="select_with_subquery_with_subquery" db-types="MySQL">
         <input sql="SELECT * FROM (select b.account_id from (select t_account.account_id from t_account) b where b.account_id=?) a WHERE account_id = 100" parameters="100" />
-        <output sql="SELECT * FROM (select b.account_id from (select t_account.account_id from t_account) b where b.account_id=?) a WHERE account_id = 100" parameters="100" />
+        <output sql="SELECT * FROM (select b.account_id from (select t_account_0.account_id from t_account_0) b where b.account_id=?) a WHERE account_id = 100" parameters="100" />
```

Comment:
@cheese8 This rewriting seems to be incorrect. The inner subquery needs to query the account_id on all shards instead of t_account_0 shard.